src/MangaJet/app/src/main/java/com/mangajet/mangajet/ui/search/SearchViewModel.kt

```diff
@@ -129,13 +127,8 @@ class SearchViewModel : ViewModel() {
         val librariesNames = Array(Librarian.LibraryName.values().size) { i ->
             Librarian.LibraryName.values()[i].resource}
         val choseResourceDialog = SearchSetSourcesDialog(librariesNames, Librarian.settings.CHOSEN_RESOURCES)
-        if (fragmentManager != null) {
+        if (fragmentManager != null)
             choseResourceDialog.show(fragmentManager, "Choose resource dialog")
-            if (choseResourceDialog.wasSelected) {
```

Comment:
Why we removed changing Librarian.settings.CHOSEN_RESOURCES[i]?

Reply:
Because we dont go in this code block. All changed states are saved in Dialog class